topology/probes/k8s/graph.go

```diff
@@ -23,35 +23,45 @@
 package k8s
 
 import (
+	"fmt"
+
 	"github.com/skydive-project/skydive/common"
 	"github.com/skydive-project/skydive/topology"
 	"github.com/skydive-project/skydive/topology/graph"
 )
 
 const (
-	manager = "k8s"
-	hostID  = ""
+	managerValue = "k8s"
+	hostID       = ""
+)
+
+const (
+	nameField      = "Name"
```

Comment:
I don't really the value of constants -especially nameField and extraField - for this

Reply:
Replaced:
- `nameField` -> `"Name"`
- `namespaceField` -> `"Namespace"`

I left `extraField` as this value is used as root for probes metadata ... a root unique for each probe ... this is changes to a standard location (shared by all probes such as `Extra`) then I agree that their is not need for the constant.